velox/exec/MergeSource.cpp

```diff
@@ -157,8 +157,9 @@ class MergeExchangeSource : public MergeSource {
           mergeExchange_->outputType(),
           &data);
 
-      mergeExchange_->stats().inputPositions += data->size();
-      mergeExchange_->stats().inputBytes += data->retainedSize();
+      auto lockedStats = mergeExchange_->stats().wlock();
```

Comment:
nit: since this code block has been used in more than once place, how define a utility say
```
void updateInputStats(xxx, yyy) {
  auto lockedStats = stats().wlock();
  lockedStats->inputPositions += xxx;
  lockedStats->inputBytes += data->retainedSize();
}

Reply:
It is used only once in MergeSource.
Do you mean adding this to the base Operator class?
This would be too specialized for a base class, I think.